src/components/home/home.component.spec.ts

```diff
@@ -1,21 +1,32 @@
 import { ComponentFixture, TestBed } from '@angular/core/testing';
 import { HomeComponent } from './home.component';
+import { ProductService } from '../../services/products/product.service';
+
 
 describe('HomeComponent', () => {
   let component: HomeComponent;
   let fixture: ComponentFixture<HomeComponent>;
+  let productService: ProductService;
```

Comment:
interesting... I see you noticed that injecting a service to consume the endpoint from backend is needed here...